### PR TITLE
update steps

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/1645

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: Exchange/ExchangeOnline/administration/how-to-change-your-user-photo.md [hunk 1]
```diff
@@ -25,19 +25,19 @@ _Original KB number:_ &nbsp; 2986893
...
-To update your user photo in Exchange Online, Lync 2013, and Lync Web App, follow these steps:
+To update your user photo in Microsoft 365, follow these steps:
 
-1. Sign in to [Microsoft 365 portal](https://portal.office.com) or [Outlook on the web](https://outlook.office.com).
+1. Sign in to [Outlook on the web](https://outlook.office.com).
 2. Select your user photo (or its placeholder) on the right side of your name.
 3. In the **My accounts** window, select your user photo or placeholder again.
-4. In the new **Account Information** window, select **Browse** to select and upload your photo.
-5. Select **Save** to set your user photo.
+4. In the appeared **Change your photo** window, select **Upload a new photo** to select and upload your photo.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/how-to-change-your-user-photo.md:33]:
```suggestion
4. In the **Change your photo** window that appears, select **Upload a new photo** to select and upload your photo.
```

File: Exchange/ExchangeOnline/administration/how-to-change-your-user-photo.md [hunk 2]
```diff
@@ -25,19 +25,19 @@ _Original KB number:_ &nbsp; 2986893
...
+5. Select **Apply** to set your user photo.
 
 ## More information
 
 In Microsoft 365, user photos are stored in the following locations:
 
 - A low-resolution photo (less than 100 KB) is stored in the user's `ThumbnailPhoto` attribute in Active Directory. This is the photo that's synchronized to Microsoft 365 in a hybrid environment. Low-resolution photos are used by Lync 2010.
-- A high-resolution photo is stored in the root directory of the user's Exchange Online mailbox. High-resolution photos are displayed in Exchange Online, Lync 2013, and Lync Web App.
+- A high-resolution photo is stored in the root directory of the user's Exchange Online mailbox. High-resolution photos are displayed in Exchange Online, Lync 2013, Skype for Business and Teams.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/how-to-change-your-user-photo.md:41]:
```suggestion
- A high-resolution photo is stored in the root directory of the user's Exchange Online mailbox. High-resolution photos are displayed in Exchange Online, Lync 2013, Skype for Business, and Microsoft Teams.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**